addons/mail/static/src/discuss/core/public_web/discuss_search.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.DiscussSearch">
+        <div t-if="tagsList.length" class="d-flex flex-wrap gap-1 m-2">
+            <TagsList tags="tagsList"/>
+        </div>
+        <div t-ref="searchBox" class="o-mail-DiscussSearch d-flex flex-grow-1 align-items-center justify-content-center rounded gap-1" t-att-class="{ 'o-active': state.searchValue.length }">
+            <input class="form-control rounded-3 border-0" t-att-placeholder="inputPlaceholder" t-model="state.searchValue" t-ref="searchInput" t-on-keydown="onKeydownInput"/>
+            <button t-if="state.searchValue.length" class="btn btn-light p-1 rounded-0" t-on-click="() => state.searchValue = ''"><i class="oi oi-close" title="Clear quick search"/></button>
+        </div>
+        <NavigableList t-props="navigableListProps"/>
+    </t>
```

Comment:
`TagsList` should be visually inside input, not above:
<img width="277" alt="Screenshot 2024-09-12 at 17 29 35" src="https://github.com/user-attachments/assets/21f52335-b189-4fa5-b97b-29e39b055247">

The technique is to give illusion that container is the input, when it actually contains input that has not borders (the style of input border is on container, which also contains tags list.